input_parser.star

```diff
@@ -238,8 +238,12 @@ DEFAULT_ROLLUP_ARGS = {
     # This flag will enable a stateless executor to verify the execution of the batches.
     # Set to true to run erigon as the sequencer.
     "erigon_strict_mode": True,
-    # Set to true to automatically deploy an ERC20 contract on L1 to be used as the gas token on the rollup.
-    "zkevm_use_gas_token_contract": False,
+    # Set to true to use an L1 ERC20 contract as the gas token on the rollup.
+    # The address of the gas token will be determined by the value of `gas_token_address`.
+    "gas_token_enabled": False,
+    # The address of the L1 ERC20 contract that will be used as the gas token on the rollup.
+    # If the address is empty, a contract will be deployed automatically.
+    "gas_token_address": "",
```

Comment:
It might be worth while to add some validation in the input_parser. E.g. if someone specifies `gas_token_address`, but leaves `gas_token_enabled` maybe the input_parser should fail with an error or something?

Reply:
Addressed by [dfcf931](https://github.com/0xPolygon/kurtosis-cdk/pull/386/commits/dfcf9315e3d54fb4627c6e42ca4b64f9c81469ec)